### PR TITLE
TW-741 : redesign quoted message

### PR DESCRIPTION
**Pull request by imGok**
Still left : add preview image thumbnail

#741 

Design : 
image

Demo : 
image

### CODE REVIEW

**Comment by hoangdat** [comment]
image

IMO, background for quote need to expand with bubble

**Comment by hoangdat** [comment]
can you demo:
- case 1: long quote
- case 2: long quote in web
- case 3: short quote in web

**Comment by imGok** [comment]
image

**Comment by imGok** [comment]
@hoangdat I increased font weight because w500 didn't look like figma, and added expanded

**Review by drminh2807 — APPROVED** [review]

File: lib/pages/chat/events/reply_content_style.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ReplyContentStyle {
+  static const double fontSizeDisplayName = AppConfig.messageFontSize * 0.76;
```
Comment (sherlockvn) [inline lib/pages/chat/events/reply_content_style.dart:6]: can you change it like the design in figma

File: lib/pages/chat/events/reply_content_style.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ReplyContentStyle {
+  static const double fontSizeDisplayName = AppConfig.messageFontSize * 0.76;
+  static const double fontSizeDisplayContent = AppConfig.messageFontSize * 0.88;
```
Comment (sherlockvn) [inline lib/pages/chat/events/reply_content_style.dart:7]: same

File: lib/pages/chat/events/reply_content_style.dart (hunk 1 above)
Comment (imGok) [reply lib/pages/chat/events/reply_content_style.dart:6]: It was implemented by fluffy, @hoangdat can you confirm we don't use `AppConfig.messageFontSize` anymore pls ?

File: lib/pages/chat/events/reply_content_style.dart [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ReplyContentStyle {
+  static const double fontSizeDisplayName = AppConfig.messageFontSize * 0.76;
+  static const double fontSizeDisplayContent = AppConfig.messageFontSize * 0.88;
+
+  static EdgeInsets replyParentContainerPadding(BuildContext context) {
```
Comment (hoangdat) [inline lib/pages/chat/events/reply_content_style.dart:9]: no need `context`

**Comment by hoangdat** [comment]
@sherlockvn please help him

**Review by hoangdat — APPROVED** [review]

**Pull request merged**